modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportG2P.pm

```diff
@@ -259,12 +258,10 @@ sub parse_input_file {
           'seq_region_start' => $gene->seq_region_start,
           'seq_region_end' => $gene->seq_region_end,
           'seq_region_strand' => $gene->seq_region_strand,
-          'mutation_consequence' => $mode,
           'inheritance_type' => $allelic,
           'pubmed_id'  => $pubmeds,
           'accessions' => \@accns,
-          'g2p_confidence' => $confidence_category,
-          'mutation_consequence' => $mutation_consequence,
+          'g2p_confidence' => $confidence_category,nce,
```

Comment:
Found the issue - this line.

Reply:
Oh that was an error in my push, fixed now. Thanks @dglemos